.github/workflows/linux-wpt.yml

```diff
@@ -65,6 +65,7 @@ jobs:
       - name: Setup Python
         uses: ./.github/actions/setup-python
       - name: Bootstrap dependencies
+        timeout-minutes: 30
```

Comment:
Let's go with 60 minutes here, just to give the jobs some margin. We can't really rely on GitHub runners always being as fast as they are now.